apps/studio/public/assets/css/preview-tw.css

```diff
@@ -1097,10 +1097,18 @@ video {
   z-index: 50;
 }
 
+.col-span-12 {
+  grid-column: span 12 / span 12;
+}
```

Comment:
i think can use css `for loop` to ease this but leave it up to you!

Reply:
this is autogenerated lol